.github/workflows/chase_release_managers.yml

```diff
@@ -0,0 +1,34 @@
+name: "Chase missing release managers in confluence"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Full version of the release to schedule (e.g. 7.31.0)'
+        required: true
+        type: string
+
+
+jobs:
+  create_release_schedule:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+      with:
+        ref: ${{ github.head_ref }}
+    - name: Install python
+      uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
+      with:
+        python-version: 3.11
+        cache: "pip"
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏There's no `pip` by default? 😳

Reply:
hm, maybe, this was generated by copilot, I didn't challenge this

Reply:
Comes with `Python 3.10` and `pip3 22.0.2`

Source: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md